docs/notebooks/expected_improvement.pct.py

```diff
@@ -1,3 +1,8 @@
+# ---
```

Comment:
@joelberkeley You don't happen to know why this isn't working? I'm trying to mark this notebook as orphan (as described in https://nbsphinx.readthedocs.io/en/0.8.3/orphan.html) so we don't get a warning that it's not part of a TOC (it's only included as a doc link in the FAQ).

Reply:
no idea. I've never worked with headers for notebooks